pkg/dispatcher/grpc/grpc.go

```diff
@@ -19,37 +19,76 @@ package grpc
 import (
 	"google.golang.org/grpc"
 
-	function "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/function"
-	fntypes "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/fntypes"
+	"github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/function"
+	_ "github.com/projectriff/function-sidecar/pkg/dispatcher/grpc/fntypes"
```

Comment:
why is this listed twice?

Reply:
Not the same package.
I'm also not 100% sure it needs to be imported if not used (hence the `_`) but there is an `init` function in there that may be required

Reply:
I'm talking about `fntypes` being listed again 5 lines below

Reply:
Ohhh... My bad